webpack.config.js

```diff
@@ -30,3 +24,33 @@ module.exports = {
     },
   },
 };
+
+const nodeConfig = {
+  ...baseConfig,
+  target: "node",
+  output: {
+    library: {
+      type: "umd",
+    },
+    filename: "index.node.js",
+    path: path.resolve(__dirname, "dist"),
+  },
+};
+
+const browserConfig = {
+  ...baseConfig,
+  target: "web",
+  experiments: {
+    outputModule: true,
+  },
+  output: {
+    library: {
+      type: "module",
+      // type: "commonjs-static",
+    },
+    filename: "index.browser.js",
```

Comment:
```suggestion
    filename: "index.mjs",
```
Again, it's not just for browsers and `mjs` tells you that it's an ES module.